libs/wingc/src/jsify.rs

```diff
@@ -722,17 +717,20 @@ impl<'a> JSifier<'a> {
 	}
 
 	fn jsify_inflight_function(&mut self, func_def: &FunctionDefinition, context: &JSifyContext) -> String {
-		let mut parameter_list = vec![];
-		for p in func_def.parameters.iter() {
-			parameter_list.push(p.0.name.clone());
-		}
-		let block = self.jsify_scope(
-			&func_def.statements,
-			&JSifyContext {
-				in_json: context.in_json.clone(),
-				phase: Phase::Inflight,
-			},
-		);
+		let parameters = func_def.parameters.iter().map(|p| &p.0.name).join(", ");
+
+		let block = match &func_def.body {
+			FunctionBody::Statements(scope) => self.jsify_scope(
+				scope,
+				&JSifyContext {
+					in_json: context.in_json.clone(),
+					phase: Phase::Inflight,
+				},
+			),
+			// TODO extern usage with closures is not supported
```

Comment:
Why is that a TODO? Do we expect this to be used for something in the future?

Reply:
I think it should be reasonable/possible to use extern for local closures. We don't need it right now, but it'd be interesting for later.

I can remove the "TODO" if you'd like, but I think the comment itself is a useful marker.